examples/fork-variable-access/fork_variable_access.metatags

```diff
@@ -0,0 +1,2 @@
+description: This sample demonstrates how the workers within a fork can access the variables that are declared in the scope of that block.
```

Comment:
Suggested edit: This Ballerina sample demonstrates how the workers within a fork can access the variables that are declared in the scope of that block.

(try to add the keywords like Ballerina in the description too)

Reply:
Add "Ballerina" before sample in all BBEs without the Ballerina keyword.